volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -866,6 +867,15 @@ def get_vma_iter(self) -> Iterable[interfaces.objects.ObjectInterface]:
         else:
             raise AttributeError("Unable to find mmap or mm_mt in mm_struct")
 
+    def get_vma_iter(self) -> Iterable[interfaces.objects.ObjectInterface]:
+        """Returns an iterator for the VMAs in an mm_struct. Automatically choosing the mmap or mm_mt as required."""
+
+        with contextlib.suppress(exceptions.InvalidAddressException):
```

Comment:
Presumably you want this inside the for loop, so it'll keep trying to return values?  I'm happy for it to bail out as soon as it hits a bad one, I'm just thinking it could potentially mask off a lot of good results as soon as one goes bad.  It also doesn't log, so the user has no way of knowing why or even that something's not quite right.  Smear is an issue, but ideally we should tell people when something's smear, rather than paper over it...